lib/rbs_rails/path_helpers.rb

```diff
@@ -1,6 +1,10 @@
 module RbsRails
   class PathHelpers
     def self.generate(routes: Rails.application.routes)
+      # Since Rails 8.0, route drawing has been deferred to the first request.
+      # This forcedly loads routes before generating path_helpers.
+      Rails.application.routes.eager_load!
```

Comment:
`RouteSet#eager_load!` was added in https://github.com/rails/rails/pull/27647 (since Rails 5.1).  So it's okay to call it simply, even if Rails 7.x or older.